### PR TITLE
Add in_progress parameter to /api/events to filter the results.

### PR DESCRIPTION
**Pull request by sergeknystautas**
Add in_progress parameter to /api/events

### CODE REVIEW

File: frigate/http.py [hunk 1]
```diff
@@ -565,6 +565,8 @@ def events():
     has_clip = request.args.get("has_clip", type=int)
     has_snapshot = request.args.get("has_snapshot", type=int)
     include_thumbnails = request.args.get("include_thumbnails", default=1, type=int)
+    in_progress = request.args.get("in_progress", default=0, type=int)
```
Comment (NickM-27) [inline frigate/http.py:568]: I think this behavior should match the other filters as in the default value is `None`, passing 0 will only return events not in progress and passing 1 will only return events in progress

**Review by NickM-27 — CHANGES_REQUESTED** [review]

File: frigate/http.py (hunk 1 above)
Comment (sergeknystautas) [reply frigate/http.py:568]: Right, I hadn't considered the other use, but the parameter would follow the same pattern.

Comment (NickM-27) [reply frigate/http.py:568]: Not sure I follow what you mean by the last part?

Comment (sergeknystautas) [reply frigate/http.py]: Sorry I dropped too many words.  I meant to say that I see your point that there's benefit of in_progress following the same pattern as has_clip and has_snapshot with possible values of none, 0, and 1.  just pushed an update that I think should work.

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**